src/ucp/wireup/select.c

```diff
@@ -664,18 +665,22 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
 static inline double
 ucp_wireup_tl_iface_latency(const ucp_worker_iface_t *wiface,
                             const ucp_unpacked_address_t *unpacked_addr,
-                            const ucp_address_iface_attr_t *remote_iface_attr)
+                            const ucp_address_iface_attr_t *remote_iface_attr,
+                            unsigned ep_init_flags)
```

Comment:
other flags are not used here, maybe just `int prioritized`, the same for all iface related functions

Reply:
wouldn't it be to specific? if we'd want to pass more EP information in the future

Reply:
specific flag improves readability, overwise it's not clear for which case it's used, better extend usage when needed, probably never :)